algorithms/estonian.sbl

```diff
@@ -0,0 +1,256 @@
+/* Estonian stemmer version 1.3
+
+Made by Linda Freienthal in January 2019.
+
+*/
+
+routines (
+	mark_regions
+	LONGV
+	special_noun_endings
+	case_ending
+	emphasis
+	plural_three_first_cases
+	remove_double_kpt
+	double
+	undouble
+	i_plural
+	degrees
+	substantive
+	verb_exceptions
+	verb
+	nu
+)
+
+stringdef a" hex 'E4' //a-umlaut ä
+stringdef o" hex 'F6' //o-umlaut ö
+stringdef o' hex 'F5' //o with tilde õ
+stringdef u" hex 'FC' //u-umlaut ü
+
+externals ( stem )
+booleans ( is_verb )
+integers ( p1 )
+groupings ( V1 RV KI GI)
+stringescapes {}
+
+define V1 'aeiou{o'}{a"}{o"}{u"}'
+define RV 'aeiuo'
+define KI 'kptgbdshfšzž'
```

Comment:
This uses literal UTF-8 accented characters, which will only work when the generated code is working in UTF-8.

You should use `stringdef` for these - add something like these above:

```
stringdef s* '{U+0161}' //s-caron š
stringdef z* '{U+017E}' //z-caron ž
```

And then `'kptgbdshf{s*}z{z*}'` here.

Reply:
Fixed by 8a0ec827a6f7017d4c6dbde0aa82d5e719d09155.